website/developers/how-to-contribute.md

```diff
@@ -106,8 +108,10 @@ $ git clone https://git-wip-us.apache.org/repos/asf/brooklyn-server.git
 
 When producing patches, please use `git format-patch` or a similar mechanism - this will ensure that you are properly
 attributed as the author of the patch when a committer merges it.
+The review process will be as with pull requests, except for comments only appearing on the Jira issue.
 
 
-## Handy Places
+### Handy Places
 
-Development bookmarks are available [here](links.html).
+If you've not done so, you'll probably want to start by [getting the code](code/).
+Once you've done that, you'll find [handy development bookmarks here](links.html).
```

Comment:
I think this section can be dropped. It duplicates other information in the side menu, and is such a small section it probably won't be noticed by many people.